packages/extension-store-locator/src/components/store-locator/store-locator-form.jsx

```diff
@@ -0,0 +1,230 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import React, {useEffect} from 'react'
+import PropTypes from 'prop-types'
+import {useForm} from 'react-hook-form'
+import {Box} from '@chakra-ui/react/dist/cjs/box/box.cjs'
+import {Button} from '@chakra-ui/react/dist/cjs/button/button.cjs'
+import {InputGroup} from '@chakra-ui/react/dist/cjs/input/input-group.cjs'
+import {Select} from '@chakra-ui/react/dist/cjs/select/select.cjs'
+import {FormControl} from '@chakra-ui/react/dist/cjs/form-control/form-control.cjs'
+import {FormErrorMessage} from '@chakra-ui/react/dist/cjs/form-control/form-error.cjs'
+import {Input} from '@chakra-ui/react/dist/cjs/input/input.cjs'
```

Comment:
This is a TODO.

There is a problem of jest not able to resolve chakra components correctly. Will fix.